docker-compose.yml

```diff
@@ -1,56 +1,21 @@
-# Provides the Feed + ElasticSearch + Kibana stack for local development. See
-# README.md for instructions on use.
+# See README.md for instructions on use.
```

Comment:
This docker-compose seems to only bring up one container. Maybe get rid of it if it does not seem useful?